# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/webrender/src/frame_builder.rs
# Commit: 5ccaf8100f79
# Full Hash: 5ccaf8100f7912e55bb98cc1951d57df9c4dd511
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-04-26 16:04:35
# Regressor Bug: 1766017
# File Overlap Count: 1
# Description:
#   Bug 1766017 - Do a better job of clipping off-screen surface allocation sizes r=gfx-reviewers,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D144534
# ==============================================================================

diff -r 01d5f09a02bf -r 5ccaf8100f79 gfx/wr/webrender/src/frame_builder.rs
--- a/gfx/wr/webrender/src/frame_builder.rs	Tue Apr 26 09:24:16 2022 +0000
+++ b/gfx/wr/webrender/src/frame_builder.rs	Tue Apr 26 09:42:22 2022 +0000
@@ -337,7 +337,7 @@
                         // re-map the dependencies of any child primitives.
                         let surface = &scene.surfaces[surface_index.0];
                         let world_culling_rect = tile_cache.pre_update(
-                            surface.local_rect,
+                            surface.unclipped_local_rect,
                             surface_index,
                             &visibility_context,
                             &mut visibility_state,